api/management/swagger-api_management.yaml

```diff
@@ -1107,6 +1107,10 @@ components:
           type: boolean
         show_account_deletion_button:
           type: boolean
+        cancel_registration_url:
+          type: string
+        confirmed_registration_url:
```

Comment:
same 
I am showing each place as, in case you decide to change the name, you see where it should be changed

Reply:
thanks